extensions/common/crypto/jws2020/src/main/java/org/eclipse/edc/security/signature/jws2020/IssuerCompat.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.security.signature.jws2020;
+
+import com.apicatalog.jsonld.JsonLd;
+import com.apicatalog.jsonld.JsonLdError;
+import com.apicatalog.jsonld.document.Document;
+import com.apicatalog.jsonld.document.JsonDocument;
+import com.apicatalog.jsonld.loader.DocumentLoader;
+import com.apicatalog.ld.DocumentError;
+import com.apicatalog.ld.signature.SigningError;
+import com.apicatalog.vc.processor.Issuer;
+import jakarta.json.Json;
+import jakarta.json.JsonObject;
+import org.eclipse.edc.util.reflection.ReflectionUtil;
+
+import java.net.URI;
+import java.util.Arrays;
+
+/**
+ * The {@link Issuer} adds the context, but currently that adds hard-coded {@code "https://w3id.org/security/suites/ed25519-2020/v1"}.
+ * For the Jwk2020 suite we need that to be {@code "https://w3id.org/security/suites/jws-2020/v1"}, so as a temporary workaround we do <em>not</em>
+ * use {@link Issuer#getCompacted()}, but rather use {@link IssuerCompat#compact(Issuer, String...)}.
+ */
+public class IssuerCompat {
```

Comment:
Typo `Compat`?

Reply:
actually no: `Compat` as in `Compatibility`

Reply:
Ok, not obvious when you are not super familiar with the code as the term `Compacted` is everywhere around

Reply:
as per @ndr-brt's [comment](https://github.com/eclipse-edc/Connector/pull/3359#discussion_r1286863934) I renamed it to `IssuerCompatibility`